app/controllers/admin/people_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
```

Comment:
The commit is adding in a controller action. A blank view is require to test that action without it throwing a template error. I was to isolate the commit to just the controller action.